### PR TITLE
Works overlay

### PR DESCRIPTION
**Pull request by lordksix**
In this PR, there were changes for mobile-size screens. The results are:

- Added a work modal functionality for screen sizes.

- Added animations for the works overlay when it pops in and pops out.
https://github.com/lordksix/portafolio-app/blob/4e7f2297828d4bdad1ab1c9f5548585e54c6f6e8/style/base_style.css#L350-L372

https://github.com/lordksix/portafolio-app/blob/4e7f2297828d4bdad1ab1c9f5548585e54c6f6e8/style/base_style.css#L394-L416

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -76,7 +77,7 @@ <h2 class="work-title">Tonic</h2>
             <li class="work-lang"><span>JavaScript</span></li>
           </ul>
           <div class="work-btns">
-            <button type="button" class="work-btn" data-workBtn="1">See project</button>
+            <button type="button" class="work-btn" data-workBtn="0">See project</button>
```
Comment (Qoosim) [inline index.html:80]: - [ ] Great job so far. However, there's one thing you need to work on before you move on to the next milestone. But believe me, you're almost there. It is required to remove all the html file handle works section and create them via array of object from your javascript file on page load. Kindly check [project requirements](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/8_popup.md) to re-confirm this requirement.
### Project Requirements
proj-rqt

**Review by Qoosim — CHANGES_REQUESTED** [review]
#  *Status: Required Changes* :recycle:

Hi @lordksix, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
-  No linter errors. :heavy_check_mark: 
-  Good README file. :heavy_check_mark: 
-  Correct GitHub Flow. :heavy_check_mark: 




## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by abdulhamiid — CHANGES_REQUESTED** [review]
Hi Team, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request ❌ ❌ ❌ 

You have submitted the wrong milestone, you have worked on the popup window but submitted them in the Form validation exercise.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: index.html
Comment (AbelG101) [inline index.html:1]: https://github.com/lordksix/portafolio-app/blob/0cfa95cfcde04a4c608d1d635aaab0472349c480/index.html#L62-L145
- [ ] Great job loading the popup dynamically but as requested by the previous reviewer the entire "works section" (not only the popup) should be removed and created dynamically with the information from the projects array when the page loads🔃. I'd advise you to check out the [project requirements](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/8_popup.md) carefully.
    - This means the above 👆 HTML code I've highlighted for the projects list should be removed, and instead, it should be added via JavaScript. 🎯
- Here's the project requirement as a reminder:
     image

File: style/base_style.css
Comment (AbelG101) [inline style/base_style.css:1]: - [ ] [OPTIONAL] The previous reviewer forgot to mention this but please make sure you stick to the Figma design for the popup window as I've noticed a small issue:
   - Your design:
    firefox_5eDZGkGxyS
    - Figma design:
      firefox_IoP67wcaUT
    - As you can see above the list of technologies and the button's border could use a fix. ✔

**Review by AbelG101 — CHANGES_REQUESTED** [review]
# Hi @lordksix 👋, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Highlights
- Correct use of GitHub Flow ✔
-  Well-structured files ✔
- All linter checks are passing ✔
- Descriptive PR title with a great summary ✔

 ## Required Changes ♻️
- _Check the comments under the review._
- You just have to make a few changes and you're on your way 🚀.

## Optional suggestions
- N/A

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by 1GORDON — APPROVED** [review]
# Status: APPROVED 🟢 

Hi @lordksix ✋

Your project is complete!  There is nothing else to say other than... it's time to merge it: ship it:
Congratulations! :tada: :clap: :clap: :100:

## Highlights!
- Followed Gitflow ✔️
- Good commit messages ✔️
- Linters are passing ✔️
- Descriptive Pull request ✔️
- Professional readme ✔️

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you consider them as they can improve your code._

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread and tag me `@1GORDON` if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**